chain/client/src/test_utils.rs

```diff
@@ -1137,7 +1145,16 @@ impl TestEnvBuilder {
     fn new(chain_genesis: ChainGenesis) -> Self {
         let clients = Self::make_accounts(1);
         let validators = clients.clone();
-        Self { chain_genesis, clients, validators, runtime_adapters: None, network_adapters: None }
+        let mut seeds: HashMap<AccountId, RngSeed> = HashMap::with_capacity(16);
+        seeds.insert(clients[0].clone(), [4; 32]);
```

Comment:
nit: maybe set this seed for all the clients? (I know that in this context we have only one)

Reply:
and in theory this is not needed, as [4;32] is the default seed if one is missing from this hashmap